api/proto/config/appconfig.proto

```diff
@@ -107,4 +107,5 @@ message AppInstanceConfig {
 message VolumeRef {
   string uuid = 1; // Volume UUID
   int64 generationCount = 2;
+  string mount_dir = 9; // Indicates volume mount point inside container
```

Comment:
How about numbering it it 3, and also adding a comment what it means when this is empty (which it will be for the root directory I assume.) Will be mount empty ones on / ? Comment should explain that.

Reply:
>How about numbering it it 3,

Missed it. Copy and paste error.
>and also adding a comment what it means when this is empty

I will add comments.
>Will be mount empty ones on / ?

No, we are mounting root directory on ```/mnt``` in initrd script.